app/services/definitions/credential_definition_publisher.py

```diff
@@ -0,0 +1,95 @@
+import asyncio
+from logging import Logger
+
+from aries_cloudcontroller import AcaPyClient
+
+from app.exceptions import CloudApiException, handle_acapy_call
+from app.services import acapy_wallet
+from app.services.revocation_registry import wait_for_active_registry
+from app.util.check_endorser_connection import check_endorser_connection
+from shared import REGISTRY_CREATION_TIMEOUT
+
+
+class CredentialDefinitionPublisher:
+    def __init__(self, controller: AcaPyClient, logger: Logger):
+        self._logger = logger
+        self._controller = controller
+
+    async def assert_public_did(self):
```

Comment:
It doesn't look like this method is called ... remember, there is now util.assert_public_did, which is what's called in services.definitions.create_cred_def:
```
    publisher = CredentialDefinitionPublisher(
        controller=aries_controller, logger=bound_logger
    )

    public_did = await assert_public_did(aries_controller)
```

Is that meant to call this publisher.assert_public_did instead? Or is one of the methods not needed anymore?

Reply:
I forgot to remove :facepalm: